core/src/lib.rs

```diff
@@ -37,7 +37,8 @@ mod proto {
     #![allow(unreachable_pub)]
     include!("generated/mod.rs");
     pub use self::{
-        envelope_proto::*, peer_record_proto::mod_PeerRecord::*, peer_record_proto::PeerRecord,
+        envelope_proto::*,
```

Comment:
> since we are on this, is there a way to impede `*` imports?

aah no, not using the `rustfmt` as far as I know. do correct me if I'm wrong.

Reply:
however there's a clippy lint config `warn-on-all-wildcard-imports` which we can set to true in the `Cargo.toml` and it will give us the warnings about these. we'd then have to fix them manually. I'd consider that a seperate issue for this PR.

Reply:
ok nice, yeah we can submit another PR with it enabled